README.md

```diff
@@ -341,16 +340,18 @@ Unit tests are written using pytest. To run the tests, use the following command
 pytest tests/
 ```
 
-This will execute all tests located in the tests directory and generate a code coverage report.
+This will execute all tests located in the tests directory.
 
 ## Code Coverage
 
 Code coverage is collected using pytest-cov coverage tool. To run the tests and collect coverage information, use the following command:
 
 ```
-pytest --cov=release_notes_generator --cov-report html tests/
+pytest --cov=release_notes_generator --cov-report=html tests/
```

Comment:
This should be `pytest --cov=. tests/ --cov-report=html`. In your version, you don't do a coverage of main script.

Reply:
I see it in request form in current code.